azavea/pfb-network-connectivity
# Neighborhood visibility flag

## Overview

Adds visibility field to Neighborhood model, so neighborhoods can be filtered to only show certain neighborhoods to the public.

NOTE: Requires rebase against #403 to work properly, so left the WIP flag for now, as if you test this branch you won't actually be able to create neighborhoods via API or admin UI 

### Demo

![screen shot 2017-05-09 at 11 37 34](https://cloud.githubusercontent.com/assets/1818302/25859248/edaad64a-34ab-11e7-8bcf-757c06c80805.png)
![screen shot 2017-05-09 at 11 37 26](https://cloud.githubusercontent.com/assets/1818302/25859247/ed8e46f6-34ab-11e7-9538-bd500f2ccaff.png)


## Testing Instructions

- Run migrations. Ensure all existing neighborhoods have default 'public' state. 
- Navigate to http://localhost:9301/#/admin/neighborhoods/create/ and create both a new public and private neighborhood
- Inspect neighborhoods at http://localhost:9200/api/neighborhoods/ to ensure visibility is correct


Closes #381
